Packs/CTM360-CyberBlindspot/Integrations/CyberBlindspot/CyberBlindspot.yml

```diff
@@ -0,0 +1,147 @@
+- additionalinfo: "Choose the direction to mirror the incident: Incoming (from CyberBlindspot to Cortex XSOAR), Outgoing (from Cortex XSOAR to CyberBlindspot), or Incoming and Outgoing (from/to Cortex XSOAR and CyberBlindspot)."
+  defaultvalue: Incoming And Outgoing
+  display: Incident Mirroring Direction
+  name: mirror_direction
+  options:
+  - None
+  - Incoming
+  - Outgoing
+  - Incoming And Outgoing
+  type: 15
+  hidden:
+  - marketplacev2
+  required: false
+- defaultvalue: 7 days
+  display: First fetch (<number> <time unit>, e.g., 12 hours. Default is `7 days`)
+  additionalinfo: The time the incidents should be fetched starting from.
+  name: first_fetch
+  required: false
+  type: 0
+- displaypassword: API Key
+  additionalinfo: The CTM360 CyberBlindspot API Key to use for fetching data.
+  name: api_key
+  required: true
+  hiddenusername: true
+  type: 9
+- name: max_fetch
+  display: Maximum Number of Incidents per Fetch
+  defaultvalue: "25"
+  type: 0
+  required: true
+  additionalinfo: Default is 25. Maximum is 200.
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Date From
+  additionalinfo: |
+    Set the date/time incidents should be fetched from (setting this will always get incidents from this date/time. Normally
+    you should not set it and rely on `First Fetch`). Format should be [%d-%m-%Y %H:%M], i.e.: '22-01-2024 13:15'
+  name: date_from
+  type: 0
+  required: false
+- display: Date To
+  additionalinfo: |
+    Set the date/time incidents should be fetched up to (setting this will always get incidents before this date/time. Normally
+    you should not set it and rely on `First Fetch`). Format should be [%d-%m-%Y %H:%M], i.e.: '22-01-2024 13:15'
+  name: date_to
+  type: 0
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+defaultclassifier: 'CTM360-CyberBlindspot Classifier'
+defaultmapperin: 'CTM360-CyberBlindspot - Incoming Mapper'
+description: 'Take action on incidents derived from CTM360 CBS threat intelligence that is directly linked to your organization.'
+display: CTM360 CyberBlindspot
+name: CTM360_CyberBlindspot
+script:
+  commands:
+  - name: ctm360-cbs-incident-list
+    description: Get the list of incidents from CBS.
+    arguments:
+    - description: Select "From" date to fetch incidents starting from it.
+      name: dateFrom
+      required: false
+    - description: Select "To" date to fetch incidents up to it.
+      name: dateTo
+      required: false
+    - description: Set number of results to fetch.
+      name: maxHits
+      required: false
+    - description: Set the order of the results.
+      name: order
+      required: false
+    outputs:
+    - contextPath: CyberBlindspot.IncidentList
+      description: List of all CBS incidents.
```

Comment:
```suggestion
    - contextPath: CyberBlindspot.IncidentList.id
      description: The incident ID.
```

Reply:
this is the way to add the context outputs. Just add more data according to the outputs from the api response.

Reply:
appreciate it, will do.

Reply:
Done in [329e3d4](https://github.com/demisto/content/pull/32381/commits/329e3d49898043c51a61083572cdc21a1f70f09f)